src/navigator-clipboard.ts

```diff
@@ -0,0 +1,26 @@
+export async function clipboardWrite(data: ClipboardItems) {
+  if (data.length === 0) return
+  if (data.length > 1) throw new Error('Support for multiple ClipboardItems is not implemented.')
```

Comment:
I'm not sure what means. Would multiple ClipBoard items for example mean that you have multiple representations of the same data (for example `text/plain` and `text/html` ...)?

Will this be a common occurrence? Is it easily avoidable in downstream code?

Reply:
This isn't defined by us, but is [per the spec](https://w3c.github.io/clipboard-apis/#typedefdef-clipboarditems). `data` must be an array of ClipboardItems.

Having said that we're doing the wrong thing here. In the steps (https://w3c.github.io/clipboard-apis/#dom-clipboard-write) it says:

> If data’s [size](https://infra.spec.whatwg.org/#list-size) is greater than 1, and the current operating system does not support multiple native clipboard items on the [system clipboard](https://w3c.github.io/clipboard-apis/#system-clipboard), then add data[0] to dataList, else, set dataList to data.

So we should just get the first item of the array instead.